shotover/src/codec/cassandra.rs

```diff
@@ -785,6 +823,17 @@ impl CassandraEncoder {
             return Ok(());
         }
 
+        if let Some(tx) = &self.stream_id_to_request_id_tx {
+            let Ok(Metadata::Cassandra(meta)) = m.metadata() else {
+                unreachable!("Gauranteed to be cassandra")
```

Comment:
Typo! "Guaranteed"

Reply:
This is a common mistake of mine so I fixed it throughout the codebase, thanks.